playbooks/ansible-cloud/appliance-base/pre.yaml

```diff
@@ -11,19 +11,9 @@
       include_role:
         name: test-setup
 
-    - name: Set SSH interface IP for appliance
-      set_fact:
-        __ssh_ip: "{{ hostvars[item].ansible_host }}"
-      with_inventory_hostnames: appliance
-
-    - name: Wait 300 seconds for port tcp/22 on appliance
-      wait_for:
-        host: "{{ __ssh_ip }}"
-        port: 22
-        search_regex: SSH
-
```

Comment:
We actually want to keep this logic, as we don't have a network check from zuul-executor. What this protects against, is if the appliance doesn't come online properly, we abort the job and reschedule it.

Reply:
If tThe the appliances are not online, the ssh-keyscan just below will fail.

Reply:
well, the appliance is online but network isn't setup. This is just a safely check, also, if sshd isn't started ssh-keyscan will fail but it will not wait until the port is open.  This logic does that

Reply:
Isn't that something already done by nodepool?

Reply:
No, this is why in fact we want it. Long term yes we want nodepool to do it for appliances, but not all support it right now. So, keeping this today should be a noop on nodepool, until all appliance are checked via nodepool

Reply:
ok, I put it back. Thanks for your patience :-)